config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder.yaml

```diff
@@ -0,0 +1,64 @@
+# periodic jobs
+
+- name: ci-kubernetes-e2e-kubeadm-kinder-external-etcd-master
+  interval: 2h
```

Comment:
@fabriziopandini preference for a lower rate - e.g. 4h? WDYT?

Reply:
I'm fine with 2h (same value for all the kubeadm jobs on master), unless there are constraints from test infra side

Reply:
I'll leave it as it is, we can tweak later if test infra requires it.